bundles/org.openhab.persistence.influxdb/src/main/feature/feature.xml

```diff
@@ -4,8 +4,9 @@
 
 	<feature name="openhab-persistence-influxdb" description="InfluxDB Persistence" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
-		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.persistence.influxdb/${project.version}</bundle>
-		<configfile finalname="${openhab.conf}/services/influxdb.cfg" override="false">mvn:org.openhab.addons.features.karaf/org.openhab.addons.features.karaf.openhab-addons-external/${project.version}/cfg/influxdb</configfile>
+		<bundle start-level="79">mvn:org.openhab.addons.bundles/org.openhab.persistence.influxdb/${project.version}</bundle>
+		<configfile finalname="${openhab.conf}/services/influxdb.cfg" override="false">mvn:org.openhab.addons.features.karaf/org.openhab.addons.features.karaf.openhab-addons-external/${project.version}/cfg/influxdb
+		</configfile>
```

Comment:
What is the reason for these changes?

Reply:
It was to start the persistence service before the bindings so they could make use of it. But the same night another pr with changes to core appeared rendering this change obsolete (when/if accepted)

Reply:
Should in fact all persistence bundles start _before_ level 80?

Reply:
Just checked, without exception all persistence bundles start at level 80.